src/types/Select.tsx

```diff
@@ -0,0 +1,5 @@
+export enum Select {
```

Comment:
The enum name 'Select' is not very descriptive. Consider renaming it to 'TodoFilter' or something more descriptive of its purpose.

Reply:
if file doesn't include jsx, no need to make extension tsx, use just ts